static/js/typeahead.js

```diff
@@ -0,0 +1,47 @@
+/**
+  Typeahead function - This function enables the typeahead input
+  and writes the investment id in #hiddenInputElement to form submit
+**/
+$(document).ready(function(){
+  $('#investment-field .typeahead').typeahead({
+    hint: true,
+    highlight: true,
+    minLength: 1
+  },
+  {
+    name: 'investments',
+    source: substringMatcher(investments)
```

Comment:
On the app.template.html look at the <script></script> block in the head of html.